### ISSUE TITLE
Active Job 

### ISSUE THREAD

**Issue by LStuker** [InWork/queue_dispatcher#6]
We should check if we can support Active Job which is part of Rails 4.2.

http://weblog.rubyonrails.org/2014/12/19/Rails-4-2-final/

http://api.rubyonrails.org/classes/ActiveJob/QueueAdapters.html

**Comment by philkman** [comment]
Did you already looked at it, what it means to support Active Job? Do we just have to get rid of the tasks and task_queues table?

**Comment by LStuker** [comment]
Not really. It looks like we have to write an adapter for queue_dispatcher. It looks like that the competitors adapter are provided by rails it self. We may have to submit to the rails project our own adapter.

Here is an example how an adapter looks like:
https://github.com/rails/rails/blob/9494c74a9d50f31831837db31ab5eeff251311ae/activejob/lib/active_job/queue_adapters/delayed_job_adapter.rb

Because I am quite busy in other projects, I don't have really time to work on this at the moment.
But it is good to know that this exist and we should talk about it if we would like to support this in feature.

**Comment by philkman** [comment]
I am not sure, that we have to submit it to the rails project. It seems like the ActiveJob-Adapters are auto loaded. I think it will be sufficient, when we provide the adapter with our gem.
